a8n: Add 3.11 changelog (#7221)
sourcegraph/sourcegraph: CHANGELOG.md

- Language statistics by commit are available via the API. [#6737](https://github.com/sourcegraph/sourcegraph/pull/6737)
- Global settings can be configured from a local file using the environment variable `GLOBAL_SETTINGS_FILE`.

> - New campaign type for removing leaked NPM credentials.

Reviewer: ```suggestion
- A new Automation campaign type allows searching for and removing leaked NPM credentials. [#6893](https://github.com/sourcegraph/sourcegraph/pull/6893)
```

- Language statistics by commit are available via the API. [#6737](https://github.com/sourcegraph/sourcegraph/pull/6737)
- Global settings can be configured from a local file using the environment variable `GLOBAL_SETTINGS_FILE`.

> - A new Automation campaign type allows searching for and removing leaked NPM credentials. [#6893](https://github.com/sourcegraph/sourcegraph/pull/6893)

Reviewer 2: ```suggestion
- A new Automation campaign type finds and removes leaked NPM credentials. [#6893](https://github.com/sourcegraph/sourcegraph/pull/6893)
```

> - Creation of Automation campaigns can now be retried in case it failed due to ephemeral errors (e.g. network problems when creating a pull request on GitHub). [#6718](https://github.com/sourcegraph/sourcegraph/issues/6718)

Reviewer 2: Does the user have to do the retrying? Or does it happen automatically? How will they know that a campaign had to be retried? 

```suggestion
- Automation campaigns can now be retried to create failed change sets due to ephemeral errors (e.g. network problems when creating a pull request on GitHub). [#6718](https://github.com/sourcegraph/sourcegraph/issues/6718)
```

- Deleting an external service will not show warnings for the non-existant service. [#5617](https://github.com/sourcegraph/sourcegraph/issues/5617)
- Suggested search filter chips are quoted if necessary. [#6498](https://github.com/sourcegraph/sourcegraph/issues/6498)
- Remove potential panic in gitserver if heavily loaded. [#6710](https://github.com/sourcegraph/sourcegraph/issues/6710)

> - Many fixes for issues with campaigns failing, workflows should be much smoother now. [#6682](https://github.com/sourcegraph/sourcegraph/pull/6682) [#6625](https://github.com/sourcegraph/sourcegraph/issues/6625) [#6658](https://github.com/sourcegraph/sourcegraph/issues/6658) [#6718](https://github.com/sourcegraph/sourcegraph/issues/6718) [#7088](https://github.com/sourcegraph/sourcegraph/issues/7088) [#6766](https://github.com/sourcegraph/sourcegraph/issues/6766) [#6717](https://github.com/sourcegraph/sourcegraph/issues/6717) [#6659](https://github.com/sourcegraph/sourcegraph/issues/6659)

Reviewer: ```suggestion
- Multiple fixes to make the preview and creation of Automation campaigns more robust and a smoother user experience. [#6682](https://github.com/sourcegraph/sourcegraph/pull/6682) [#6625](https://github.com/sourcegraph/sourcegraph/issues/6625) [#6658](https://github.com/sourcegraph/sourcegraph/issues/6658) [#6718](https://github.com/sourcegraph/sourcegraph/issues/6718) [#7088](https://github.com/sourcegraph/sourcegraph/issues/7088) [#6766](https://github.com/sourcegraph/sourcegraph/issues/6766) [#6717](https://github.com/sourcegraph/sourcegraph/issues/6717) [#6659](https://github.com/sourcegraph/sourcegraph/issues/6659)
```

- Deleting an external service will not show warnings for the non-existant service. [#5617](https://github.com/sourcegraph/sourcegraph/issues/5617)
- Suggested search filter chips are quoted if necessary. [#6498](https://github.com/sourcegraph/sourcegraph/issues/6498)
- Remove potential panic in gitserver if heavily loaded. [#6710](https://github.com/sourcegraph/sourcegraph/issues/6710)
- Multiple fixes to make the preview and creation of Automation campaigns more robust and a smoother user experience. [#6682](https://github.com/sourcegraph/sourcegraph/pull/6682) [#6625](https://github.com/sourcegraph/sourcegraph/issues/6625) [#6658](https://github.com/sourcegraph/sourcegraph/issues/6658) [#7088](https://github.com/sourcegraph/sourcegraph/issues/7088) [#6766](https://github.com/sourcegraph/sourcegraph/issues/6766) [#6717](https://github.com/sourcegraph/sourcegraph/issues/6717) [#6659](https://github.com/sourcegraph/sourcegraph/issues/6659)

> - Changing an external service configuration to remove a repository won't lead to problems with the syncing process anymore even if the repository was referenced in an Automation campaign. [#7015](https://github.com/sourcegraph/sourcegraph/pull/7015)

Reviewer 2: ```suggestion
- Repositories referenced in Automation campaigns that are removed in an external service configuration change won't lead to problems with the syncing process anymore. [#7015](https://github.com/sourcegraph/sourcegraph/pull/7015)
```
